kedro/templates/project/cookiecutter.json

```diff
@@ -3,5 +3,6 @@
     "repo_name": "{{ cookiecutter.project_name.strip().replace(' ', '-').replace('_', '-').lower() }}",
     "python_package": "{{ cookiecutter.project_name.strip().replace(' ', '_').replace('-', '_').lower() }}",
     "kedro_version": "{{ cookiecutter.kedro_version }}",
-    "add_ons": "none"
+    "add_ons": "none",
+    "example_pipeline": "No"
```

Comment:
It looks inconsistent to me that the default for `add_ons` is `none` with a lower case and for `example_pipeline` it's `No` with a capital letter. I'd just use `no` without capital letter.